src/styles/main.scss

```diff
@@ -1,3 +1,112 @@
+html {
+  font-family: Roboto, sans-serif;
+  --main-color: #00acdc;
+  --text-color: #000;
+  --back-color: #fff;
```

Comment:
Move all variables to the :root selector, it is better practice.

Reply:
not fixed